src/components/DocumentationTopic.vue

```diff
@@ -409,10 +417,14 @@ export default {
     shouldRenderTopicSection: ({
       topicSectionsStyle,
       topicSections,
-    }) => topicSections && topicSectionsStyle !== TopicSectionsStyle.hidden,
+      enableMinimized,
+    }) => topicSections && topicSectionsStyle !== TopicSectionsStyle.hidden && !enableMinimized,
     isOnThisPageNavVisible: ({ topicState }) => (
       topicState.contentWidth > ON_THIS_PAGE_CONTAINER_BREAKPOINT
     ),
+    enableMinimized({ $route }) {
```

Comment:
I suggest that we make a `minimal` prop for this instead of creating a computed property that checks the route.

In the actual routing config, we can map a URL path to a component/view that then sets this prop.